index.js

```diff
@@ -500,45 +517,24 @@ RingPop.prototype.lookupN = function lookupN(key, n) {
 };
 
 RingPop.prototype.reload = function reload(file, callback) {
```

Comment:
I'm not sure if/how we want to support this function. It's currently called from the `admin/reload`-endpoint but only supports a file.